core/templates/dev/head/css/oppia.css

```diff
@@ -2851,7 +2851,7 @@ div#ng-curtain {
 .nav > li > div.oppia-navbar-tab {
   font-family: "Capriola", "Roboto", Arial, sans-serif;
   font-size: 14px;
-  padding: 17px 12px 10px 12px;
+  padding: 17px 12px 10px 12px ;
```

Comment:
Please check the diff at https://github.com/oppia/oppia/pull/3817/files and ensure there are no superfluous changes before submitting a PR for review. The diff should reflect exactly the changes you want to make, and no more.